dietpi/dietpi-software

```diff
@@ -8512,10 +8512,14 @@ _EOF_
 		then
 			G_CONFIG_INJECT 'do-ip6:[[:blank:]]' '    do-ip6: no' /etc/unbound/unbound.conf.d/dietpi.conf
 		fi
-		if grep -q 'CONFIG_PREFER_IPV4=0' /boot/dietpi.txt
+		elif grep -q 'CONFIG_PREFER_IPV4=0' /boot/dietpi.txt
 		then
 			G_CONFIG_INJECT 'prefer-ip6:[[:blank:]]' '    prefer-ip6: yes' /etc/unbound/unbound.conf.d/dietpi.conf
 		fi
+
+		if (( $G_DISTRO == 4 )); then
+			G_CONFIG_INJECT 'log-replies: no' '    # log-replies: no'
```

Comment:
Let's simply do `sed -i '/log-replies/d' /etc/unbound/unbound.conf.d/dietpi.conf` (don't forget the file name). A commented entry (and that it is printed to console) might leave the false impression that this setting can be used.